project/src/main/puzzle/tutorial/tutorial-squish-module.gd

```diff
@@ -17,14 +17,11 @@ var _show_diagram_count := 0
 ## set of level IDs which the player has attempted during this tutorial
 ## key: level id
 ## value: true
-var _prepared_levels: Dictionary
+var _level_attempt_count: Dictionary
```

Comment:
Update this comment.

Reply:
Updated to `number of attempts, initialized to '1' during the player's first attempt`